packages/noya-state/src/reducers/applicationReducer.ts

```diff
@@ -224,6 +226,13 @@ export function applicationReducer(
     case 'goToSymbolSource':
     case 'setOverrideValue':
       return symbolsReducer(state, action);
+    case 'setScale':
```

Comment:
We might want to call these

`setExportScale`
`setExportName`
`setExportFileFormat`
`setExportNamingScheme`

Since we have a lot of other things that `setName` and even the others could refer to